docker-compose.yml

```diff
@@ -23,6 +23,9 @@ x-environment:
     LAGOON_ROUTE: &default-url ${LOCALDEV_URL:-http://govcms8-saas.docker.amazee.io}
     DEV_MODE: ${DEV_MODE:-false}
     X_FRAME_OPTIONS: ${X_FRAME_OPTIONS:-SameOrigin}
+    DRUPAL_SHIELD_USER: ${DRUPAL_SHIELD_USER:-govcms}
```

Comment:
Currently this can't be overwritten for local development, because docker compose seems to ignore `DRUPAL_SHIELD_USER=` set in env, or it's just getting overwritten by `${DRUPAL_SHIELD_USER:-govcms}`

Reply:
`.env` inheritance is frustrating, that _should_ use an env value first but I think it might supersede the `.env` file so it's always defaulting. I'm going to remove the default values here so it shouldn't be a problem going forward.

Reply:
Very frustrating. And how .env is used differently by every tool...